docs/installing.md

```diff
@@ -105,7 +105,7 @@ curl -L https://raw.githubusercontent.com/kubeshop/tracetest/main/install-cli.sh
 ### MacOS
 
 ```sh
-curl -L https://raw.githubusercontent.com/kubeshop/tracetest/main/install-cli.sh | sh
+brew install kubeshop/tracetest/tracetest
```

Comment:
Don't we need to add the brew tap before running that?

Reply:
I think that's required if you don't follow the naming convention. Running just that command works fine, at least with the current version